blocked-edges/4.19.0-ec.0-VsphereStorageMountIssues.yaml

```diff
@@ -0,0 +1,12 @@
+to: 4.19.0-ec.0
+from: 4[.]18[.].*
+url: https://issues.redhat.com/browse/STOR-2486
+name: VsphereStorageMountIssues
+message: Vsphere customers using vSAN file volumes can't mount vSphere shared volumes and nfs volumes which server do not set NFS4ERR_ATTRNOTSUPP
```

Comment:
Nit: "Vsphere" -> "VSphere" in the `name` (to match `VSphereHW14CrossNodeNetworkingError` and `VSphereNodeNameChanges`) and "vSphere" in the `message`.  Having both `Vsphere` and `vSphere` in the `message` seems confusing, and [Broadcom seems to prefer "vSphere"][1], even as the opening word in a sentence or title:

![image](https://github.com/user-attachments/assets/02c3e63c-54ba-4c38-a5d8-523973905269)


[1]: https://techdocs.broadcom.com/us/en/vmware-cis/vsphere.html

Reply:
Another nit: [NFS is an acronym][1], so you probably want "nfs" -> "NFS" in this `message`.  I'm also not seeing any NFS discussion in [STOR-2486][2], just the mention in the title.  Maybe we don't need to talk about NFS volumes at all, and it's just the vSAN volumes?  Maybe NFS volumes and vSAN volumes are the same thing?  I'm not sure.

[1]: https://en.wikipedia.org/wiki/Network_File_System
[2]: https://issues.redhat.com/browse/STOR-2486

Reply:
just took the words from the ticket